apps/src/code-studio/assets/SpriteUpload.js

```diff
@@ -1,31 +1,48 @@
 import React from 'react';
 import color from '@cdo/apps/util/color';
 
+const spriteLocation = {
```

Comment:
nit: we usually start enums with a capital letter like this:
https://github.com/code-dot-org/code-dot-org/blob/47be99d6cf7df2be746b592906f50c0f3860b80a/apps/src/lib/ui/accounts/constants.js#L4-L8

we could also use the `makeEnum` util function here!